tests/core/lcp/test_credential.py

```diff
@@ -26,13 +30,7 @@ def setup_method(self):
                 LCPCredentialType.PATRON_ID.value,
                 "get_patron_id",
                 "52a190d1-cd69-4794-9d7a-1ec50392697f",
-            ),
-            (
-                "get_patron_passphrase",
-                LCPCredentialType.LCP_PASSPHRASE.value,
-                "get_patron_passphrase",
-                "52a190d1-cd69-4794-9d7a-1ec50392697f",
-            ),
```

Comment:
... Because I forgot to commit the replacement. Essentially that particular case couldn't be sanely generated with parameterization anymore, because the type involved was no longer a raw `str`. Commits incoming shortly...